consensus/src/execution_pipeline.rs

```diff
@@ -125,6 +125,7 @@ impl ExecutionPipeline {
         tokio::task::spawn_blocking(move || {
             let txns_to_execute =
                 Block::combine_to_input_transactions(validator_txns, input_txns.clone(), metadata);
+            // TODO: next version remove signature verification, it's done in quorum store
```

Comment:
I'm generally very scared of this change without any type safety that differentiate verified txn, the comment seems confusing too for direct mempool or inlined batches

Reply:
+1, may be we can change the type to `SignatureCheckedTransactions`  in next version?